x-pack/plugins/synthetics/public/legacy_uptime/components/overview/monitor_list/columns/test_now_col.tsx

```diff
@@ -82,7 +82,8 @@ export const TEST_NOW_AVAILABLE_LABEL = i18n.translate(
 export const PRIVATE_AVAILABLE_LABEL = i18n.translate(
   'xpack.synthetics.monitorList.testNow.available.private',
   {
-    defaultMessage: 'For now, Test now is disabled for private locations monitors.',
+    defaultMessage:
+      'Test now is currently not available for monitors running on Private Locations.',
```

Comment:
@justinkambic do you have a screenshot of how this is being used? My initial reaction is that _Test now ..._ is confusing, but if this tooltip is targeting a button that says "Test now" it might be ok (or we could just say "This is not currently available for...").

Reply:
@colleenmcginnis if the monitor in the list is _not_ fleet-managed and has a valid config ID, we render a [`play` button](https://github.com/elastic/kibana/blob/0283f48b543211dc3753e1acd6d9db59f48bec23/x-pack/plugins/synthetics/public/legacy_uptime/components/overview/monitor_list/columns/test_now_col.tsx#L58) which will kick off the one-off test.

We render the tooltip that uses this label in the case that the monitor is fleet-managed. So if the user created the monitor with the synthetics integration, and it's being handled with elastic agent/fleet rather than the synthetics service, we're not able to test it on demand.

I don't have a screenshot of this and my env isn't configured in a way that I can easily get one, but does that explanation make sense? If a screenshot is still necessary I can see about getting one for you today.

Reply:
Thanks @justinkambic. What to you think about something like this instead?

>You can't currently test monitors running on private locations on demand.

Reply:
@colleenmcginnis I am attaching screenshots of the three states this play (Test now) button can have. Please see if 

> You can't currently test monitors running on private locations on demand.

is till valid observing the context in screenshots. Also, we may need to change the copy for the other disabled state similar to this one.

When **Test now** is available.
![Screenshot 2022-08-10 at 17 16 53](https://user-images.githubusercontent.com/2748376/183944880-cf9249b5-10e2-48a2-ab3b-1a1667caa271.png)

When **Test now** is not available due to monitor running on Private Loations.
![Screenshot 2022-08-10 at 17 16 56](https://user-images.githubusercontent.com/2748376/183944959-2509dd8e-e56a-4247-9f7c-3f4451844de6.png)

When **Test now** is not available due to a legacy monitor.
![Screenshot 2022-08-10 at 17 17 02](https://user-images.githubusercontent.com/2748376/183944982-dd4fb279-2c2f-4a7d-9514-c841c63731fd.png)